modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -126,11 +130,13 @@ after the execution of the scope:
 </flow>
 ----
 
-In all these cases the transaction created at Flow level will be rolled back, and the message will be available to be read again. We analyze what happens with the `try` Scope:
+The behavior in the `try` Scope is:
+
+* Ignore (`INDIFFERENT`): The transaction is _continued_ but the error is not handled by an `on-error-continue` at the flow level. This causes the transaction to be rolled back, and the message is not sent.
+* Always Begin (`ALWAYS_BEGIN`): This raises an error because an active transaction already exists.
+* Begin or Join (`BEGIN_OR_JOIN`): Same behavior as `INDIFFERENT`.
```

Comment:
DITTO all previous edits for the same text.

"The transaction continues, but the error..., causing the transaction to be rolled back and the message to be unsent."

"Raises an error...."

"Displays the same behavior...."